ios/FluentUI/Command Bar/CommandBarCommandGroupsView.swift

```diff
@@ -18,7 +18,7 @@ class CommandBarCommandGroupsView: UIView {
 
         buttonGroupsStackView.translatesAutoresizingMaskIntoConstraints = false
         buttonGroupsStackView.axis = .horizontal
-        buttonGroupsStackView.spacing = LayoutConstants.buttonGroupSpacing
+        buttonGroupsStackView.spacing = (UIDevice.current.userInterfaceIdiom == .phone) ? LayoutConstants.buttonGroupSpacingPhone : LayoutConstants.buttonGroupSpacingPad
```

Comment:
Seems like we should be using size classes rather than device interface idiom here.

Reply:
I made the change to use size classes locally but forgot to amend it to the commit 🤦‍♂️. Will update.